Dockerfile

```diff
@@ -64,7 +64,7 @@ RUN yum update -y && \
     JAVA_BIN_PATH=$(rpm -ql java-${JAVA_PKG_VERSION}-openjdk-${PKG_DEVEL:-headless} | grep '\/bin\/java$') && \
     test -L $JAVA_HOME || ln -sf ${JAVA_BIN_PATH%*/bin/java} $JAVA_HOME
 
-FROM rockylinux:8.5@sha256:5fed5497b568bcf7a90a00965987fc099edbcf44b1179a5ef6d4b47758281ca5 AS rockylinux8
+FROM rockylinux:8.5.20220308@sha256:c7d13ea4d57355aaad6b6ebcdcca50f5be65fc821f54161430f5c25641d68c5c AS rockylinux8
```

Comment:
https://hub.docker.com/_/rockylinux?tab=tags

they didn't update 8.5 tag, so dependabot is right

don't know if we should switch to `:8`

<img width="1294" alt="Screenshot 2022-03-16 at 17 03 46" src="https://user-images.githubusercontent.com/71768/158634220-08b13e95-0a65-44e8-ad21-2e73545ff849.png">

Reply:
what shall we do? merge this or put 8 instead that looks updated correctly?

Reply:
I think we should stick to what dependabot wants to do here. it will most likely also pickup next 8.5.20... which is fine